cpp/src/parquet/page_index.h

```diff
@@ -126,4 +130,75 @@ class PARQUET_EXPORT OffsetIndex {
   virtual const std::vector<PageLocation>& page_locations() const = 0;
 };
 
+/// \brief Interface for reading the page index from a Parquet row group.
```

Comment:
```suggestion
/// \brief Interface for reading the page index for a Parquet row group.
```
the previous phrasing makes it sound like page index is stored within rowgroups (at least to me)